### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by rivalez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
 In order to run server you need to use script runServer.sh (./runServer.sh).
-In order to run client there has to be currently running server of battleship game and then run runClient.sh script.
\ No newline at end of file
+In order to run client there has to be currently running server of battleship game and then run runClient.sh script.
```
Comment (robwrobel) [inline README.md]: maybe retranslate to "In order to run client make sure that battleship game server is currently running and then run runClient.sh script"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+
+Instruction
+
+At the beginning window you should provide IP (in version-4) of server you wish to connect. 
+
+At the beginning of the game you need to place your ships in following order: one 4-mast ship, two 3-mast ships, three 2-mast ships, four 1-mast ship.
+You can choose orientation of ship:
+if you click right mouse button on cell it will put horizontally ship beginning at this place and the rest of the ship will be on the right side of the begging point,
```
Comment (robwrobel) [inline README.md]: begining not begging?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
+if you click left mouse button on cell it will put vertically ship beginning at this place and the rest of the ship will be below of the begging point.
```
Comment (robwrobel) [inline README.md]: begining not begging?

File: README.md (hunk 1 above)
Comment (rivalez) [reply README.md]: looks fine for me

**Pull request merged**